content/30-use-pah/_index.md

```diff
@@ -7,106 +7,131 @@ weight = 30
 
-#### Sync collections from Red Hat Automation Hub
+#### Sync collections from Red Hat Automation Hub to PAH
 
 * Go to `console.redhat.com` and open **Ansible Automation Platform** -> **Automation Hub** -> **Collections**. Here you can enable/disable the sync of certain collections.
-
 * What you need to do is to get the authentication token and configure it in your PAH:
-
   * In Red Hat Automation Hub go to **Connect to Hub** we will need the **Offline Token** and the **Server URL**
-
   * In your private automation hub go to **Collections** -> **Repository management** -> **Remote**
-
   * Edit the `rh-certified` remote:
-
     * **URL** paste the **Server URL** from the Red Hat Automation Hub
-
     * **Token** the token you copied from RH AH
-
     * Click **Save** and then hit **Sync**. This will sync all enabled and updated collections from Red Hat Automation Hub to your Private Automation Hub.
+* After the sync process has finished, check the certified collections are visible on PAH.
 
-#### Sync selected community collections from Ansible Galaxy
+#### Integrate your PAH in Automation Controller
 
-Galaxy is configured as the remote `community` out of the box. Follow the instructions to configure the synchronization.
+Automation Controller doesn't know about your Private Automation Hub, yet. The first step is to create a new Credential pointing to your PAH:
 
-* Create a regular requirements.yml file pointing to the collection you want, we'll use the one from the docs:
+Get access URL and Token fromm your PAH and put them down somewhere:
 
-```yaml
-collections:
-  # Install a collection from Ansible Galaxy.
-  - name: containers.podman
-```
+* In PAH, **Collections**->**Repository management**
+* Get the URLs for the **local** repos **community**, **published** and **rh-certified**
+* Get the access token from **Collections**->**API Token**
 
-* Go to Repo Management, click the **Remote** tab again
+Then configure three new credentials in Automation Controller for the three URLs:
 
-* Edit the `community` remote
+* **Name**: PAH {Community,Published,RH-Certified}
+* **Organization**: default
+* **Credential Type**: Ansible Galaxy/Automation Hub API Token
+* **Galaxy Server URL**: <the respective community repo URL>
+* **API Token**: <the token>
 
-* In **YAML requirements** upload the  `requirements.yml` file from your local machine.
+Now the access information is configured but not used by Automation Controller. This is done on **Organization** level.
 
-* Click **Save**
+* Open the `default` **Organization** in Automation Controller and click **Edit**
+* Have a look at the **Galaxy Credentials** field. There is only one credential pointing to Ansible Galaxy. This is the default.
+* Remove the credential and add the one pointing to your PAH.
```

Comment:
... and add the three credentials you just created.

Reply:
Also "click **Save**"

Reply:
Here is also a step missing to disable SSL Certificate check or the project import will fail during "fetch galaxy collections" task.
Disable SSL verification in **Settings** -> **Job Settings**, Edit and turn on **Ignore Ansible Galaxy SSL Certificate Verification**

Reply:
ALSO: the community collection is not configured - the chapter from below "Sync selected community collections from Ansible Galaxy to PAH" has to move up